Gopkg.toml

```diff
@@ -4,29 +4,33 @@
   go-tests = true
   unused-packages = true
 
-[[constraint]]
+[[override]]
   name = "github.com/asecurityteam/settings"
-  branch = "master"
+  version = "^0.1.0"
 
 [[constraint]]
   name = "github.com/asecurityteam/serverfull"
-  version = "0.1.0"
+  version = "^0.4.0"
 
 [[constraint]]
-  name = "github.com/asecurityteam/serverfull-gateway"
-  branch = "master"
+  name = "github.com/asecurityteam/component-producer"
+  version = "^0.4.0"
```

Comment:
Nit: we should probably bump the producer version to ^0.4.1 and run `dep ensure -update` to grab the 2xx handling fix, just in case.